### PR TITLE
Create workflow for manual production deploys

### PR DESCRIPTION
**Pull request by dhruvkb**
## Description

This PR creates a workflow 'Deploy ECS' that takes an image tag and deploys that to production for either frontend or API (not yet developed).

## Checklist

- [x] My pull request has a descriptive title (not a vague title like
      `Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or
      a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible
      errors.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: .github/workflows/prod_deploy.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+name: Deploy ECS
+
+on:
+  workflow_dispatch:
+    inputs:
+      stack:
+        type: choice
+        descriptions: The layer of the stack to deploy
+        options:
+          - api
```
Comment (zackkrida) [inline .github/workflows/prod_deploy.yml:10]: Should we comment this out as well as a `todo`, until the action exists?

**Review by zackkrida — APPROVED** [review]
Great. It's nice to have the API scaffolded out too.

File: .github/workflows/prod_deploy.yml (hunk 1 above)
Comment (dhruvkb) [reply .github/workflows/prod_deploy.yml]: Done in 4f6db59.

**Comment by AetherUnbound** [comment]
We sort of already have this by way of the rollback action: https://github.com/WordPress/openverse/blob/main/.github/workflows/rollback.yml

I'm also working on augmenting this as part of https://github.com/WordPress/openverse-infrastructure/pull/378

Could we use the existing workflow instead of creating a new one? Or combine these in some way?

**Comment by dhruvkb** [comment]
That's a good point @AetherUnbound. I was unaware of how rollbacks work before this. I'll close this PR and merge these workflows.

**Pull request closed without merging**